src/miniAudicle.h

```diff
@@ -157,7 +163,9 @@ class miniAudicle
     t_CKBOOL set_named_chugins( std::list< std::string > & chugins );
     t_CKBOOL get_named_chugins( std::list< std::string > & chugins );
     t_CKBOOL add_query_func(t_CKBOOL (*func)(Chuck_Env *));
-    
+    // 1.5.0.1 (ge) added; takes an RtAudio::Api enum
+    t_CKBOOL set_driver(t_CKUINT driver);
```

Comment:
style note: here `set_driver` takes a `t_CKUINT` while `probe` takes a `const char *` -- preferable to consolidate to one or the other

also should add a comment where one can find acceptable values for each/either of these -- e.g. `ChuckAudio::getAvailableApis` or something